### DISCUSSION TITLE
Pyinstaller error

### DISCUSSION THREAD

**Discussion by chickenburge** [Project-MONAI/MONAI#6995] [Q&A]
Entered this piece of code:：

import monai
print(monai.__version__)

-----------------------------------------------------------------------------------------------------------------

Next, use pyinstaller to package it as an exe. file, with code (pyinstaller -F [FileName]),  but indicate an error after packaging: 

Multiple versions of MONAI may have been installed?
Please see the installation guide: https://docs.monai.io/en/stable/installation.html

[26880] Failed to execute script FileName' due to unhandled exception!
-----------------------------------------------------------------------------------------------------------------

**Comment by KumoLiu** [comment]
Hi @chickenburge, perhaps this [discussion](https://github.com/Project-MONAI/MONAI/discussions/4197) can help you.

Moving this one to discussion for now, feel free to recreate another issue, thanks!
